#!/bin/sh -e
## 40-encoding-correction.dpatch by Luca Capello <luca@pca.it>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Remove the now useless Encoding key

if [ $# -lt 1 ]; then
echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
      -patch) patch $patch_opts -p1 < $0;;
      -unpatch) patch $patch_opts -p1 -R < $0;;
      *)
      echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
      exit 1;;
esac

exit 0

@DPATCH@
--- foo2zjs-20081007dfsg/hplj1020.desktop.ORG	2008-10-18 01:20:33.000000000 +0200
+++ foo2zjs-20081007dfsg/hplj1020.desktop	2008-10-18 02:26:13.000000000 +0200
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Encoding=UTF-8
 Name=HPLJ 10xx Replaced Paper
 Comment=HP LaserJet 1018, 1020
 zExec=usb_printerid /dev/usb/lp0
